Dockerfile.aarch64

```diff
@@ -92,7 +92,8 @@ RUN \
     /config \
     /defaults \
     /lsiopy && \
-  echo "**** cleanup ****" && \
+    printf "Linuxserver.io version: ${VERSION}\nBuild-date: ${BUILD_DATE}" > /build_version && \
```

Comment:
this means we have to remember to add this to all downstream images we rebase to 3.20? Otherwise they'll display the baseimage's version

Reply:
Yeah, the other option is not to include it on the base images, which might be a better option

Reply:
yeah, we really need to see the version of the downstream image but not so much in the baseimage